<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetDatabase</class>
 <widget class="QWidget" name="WidgetDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="gbDatabase">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>551</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Database</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="lDatabaseFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Database file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDatabaseFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbNew">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>New...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOpen">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbStop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbSettings">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>551</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QLabel" name="lPeriod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Period (maximum):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbPeriod">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>86399</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
